austation/code/modules/clothing/under/jobs/medisci.dm

```diff
@@ -46,6 +46,18 @@
 	can_adjust = FALSE
 	fitted = FEMALE_UNIFORM_TOP
 
+/obj/item/clothing/under/rank/rnd/roboticist/skirt
+	name = "roboticist's jumpskirt"
+	desc = "It's a slimming black with reinforced seams; great for industrial work."
+	icon = 'austation/icons/obj/clothing/uniforms.dmi'
+	alternate_worn_icon = 'austation/icons/mob/uniform.dmi'
+	icon_state = "robotics_skirt"
+	item_state = "robotics"
+	item_color = "robotics_skirt"
+	body_parts_covered = CHEST|GROIN|ARMS
+	can_adjust = FALSE
+	fitted = FEMALE_UNIFORM_TOP
+
```

Comment:
this goes in engineering.dm, not medsci. Yeah I know it's weird but that's how normal robo is done

all you have to do is remove the /engineering, the "rnd" is not needed

original code:
`/obj/item/clothing/under/rank/roboticist`

Reply:
it wouldn't work like that as the skirt varient has to be a subclass of the original varient, and the original varient is written as
`/obj/item/clothing/under/rank/rnd/roboticist/skirt`

Reply:
I am aware, I was quoting the original robo jumpsuit code. Look at the original code for the skirt variant in engineering.dm. This code would work too but I'm just trying to keep things consistent :)

Reply:
Well, to clear up, there are two roboticist's jumpsuit definitions: `/obj/item/clothing/under/rank/rnd/roboticist` (rnd.dm) and `/obj/item/clothing/under/rank/roboticist` (engineering.dm). Def without `rnd` in it is considered *obsolete*, so making a new jumpskirt def with `rnd` actually is the way; and can you remove obsolete definitions to clarify things?

Reply:
Oh weird. Yeah just comment out the engineering.dm one then :P

Reply:
*SS13 shitcode strikes again*